README.md

```diff
@@ -2,29 +2,13 @@
 ![Run tests](https://github.com/chemellia/ChemistryFeaturization.jl/workflows/Run%20tests/badge.svg) [![codecov](https://codecov.io/gh/chemellia/ChemistryFeaturization.jl/branch/main/graph/badge.svg?token=C0Fdt8BGnr)](https://codecov.io/gh/chemellia/ChemistryFeaturization.jl)  [![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://chemistryfeaturization.chemellia.org/stable/)
  [![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://chemistryfeaturization.chemellia.org/dev/)
 
-Flexible, modular, invertible featurization for Chemellia models including AtomicGraphnets.jl and WeaveModel.jl.
+Interface for modular, flexible, invertible featurization of atomic structures for machine learning purposes.
 
-[Check out our JuliaCon 2021 talk here!](https://www.youtube.com/watch?v=la9asuZzVjU)
+[Check out our JuliaCon 2021 talk here!](https://www.youtube.com/watch?v=la9asuZzVjU) (note that some syntax is out of date)
```

Comment:
Might be useful to mention with what version of `ChemistryFeaturization` that syntax would still hold too.

Reply:
It's noted in the video itself (I'm glad I remembered to add that in!) so I'm not sure it's crucial here?